src/main/java/org/cryptomator/integrations/tray/TrayMenuController.java

```diff
@@ -30,6 +30,13 @@ static Optional<TrayMenuController> get() {
 	 */
 	void showTrayIcon(byte[] imageData, Runnable defaultAction, String tooltip) throws TrayMenuException;
 
+	/**
+	 * Updates an icon on the system tray.
```

Comment:
```suggestion
	 * Updates the icon on the system tray.
```

We don't want to update any arbitrary tray icon. 😉

Reply:
Done in https://github.com/cryptomator/integrations-api/pull/14/commits/c7ecab8105edad15f4fde22b343282a6c31d2e12